packages/ERTP/src/issuerKit.js

```diff
@@ -15,6 +15,16 @@ import './types.js';
 
 /**
  * @template {AssetKind} K
+ * @typedef {object} IssuerRecord
+ * @property {string} name
+ * @property {K} assetKind
+ * @property {AdditionalDisplayInfo} displayInfo
+ * @property {Pattern} elementShape
+ */
+
+/**
+ * @template {AssetKind} K
+ * @param {IssuerRecord<K>} issuerRecord
  * @param {Baggage} issuerBaggage
  * @param {ShutdownWithFailure=} optShutdownWithFailure If this issuer fails
```

Comment:
```suggestion
 * @param {ShutdownWithFailure | undefined} optShutdownWithFailure If this issuer fails
```
If this parameter here is not longer optional, in the sense that it may be omitted, but still may be `undefined`, then I prefer that you type it as ` | undefined`.